dev/com.ibm.websphere.appserver.features/visibility/public/cdi-2.0/com.ibm.websphere.appserver.cdi-2.0.feature

```diff
@@ -26,7 +27,10 @@ IBM-API-Package: javax.decorator;  type="spec", \
  org.jboss.weld.proxy; type="internal", \
  org.jboss.weld.security; type="internal", \
  org.jboss.weld.serialization.spi; type="internal", \
- org.jboss.weld.context;type="third-party", \
+ org.jboss.weld;type="internal",\
```

Comment:
Why are these needed?  We specifically removed these in cdi-1.2 because of a classloading conflict reported by a user.

Reply:
That is needed for the IBMViewHandler to work, otherwise we get CNFEs. I tried to get it to work by exporting to the TCCL but that did not seem to work.

Reply:
Then we should have the `jsfContainer-2.3` feature expose the Weld packages it needs -- that way the `cdi-2.0` feature doesn't always leak these weld packages to the app classloader.